qpdf/qtest/qpdf/issue-143.out

```diff
@@ -14,9 +14,9 @@ WARNING: issue-143.pdf (object 1 0, offset 24): expected dictionary key but foun
 WARNING: issue-143.pdf (object 1 0, offset 21): stream dictionary lacks /Length key
 WARNING: issue-143.pdf (object 1 0, offset 84): attempting to recover stream length
 WARNING: issue-143.pdf (object 1 0, offset 84): recovered stream length: 606
-WARNING: issue-143.pdf object stream 1 (object 0 0, offset 0): object id is invalid
-WARNING: issue-143.pdf object stream 1 (object 0 0, offset 0): object id is invalid
-WARNING: issue-143.pdf object stream 1 (object 6 0, offset 0): offset is invalid (must be larger than previous offset 0)
-WARNING: issue-143.pdf object stream 1 (object 0 0, offset 0): object id is invalid
+WARNING: issue-143.pdf object stream 1 (object 0 0, offset 4): object id is invalid
+WARNING: issue-143.pdf object stream 1 (object 0 0, offset 15): object id is invalid
```

Comment:
@jberkenbilt  , what offset should we be using here? The position where the invalid entry is at the beginning of the object stream, or the offset where the invalid entry claims the object is located?

Reply:
@m-holger  I'd probably go with the position of the invalid entry at the beginning of the object stream.